src/js/labels/reducer.js

```diff
@@ -1,25 +1,14 @@
 import { createReducer } from "@reduxjs/toolkit";
-import { CREATE_LABEL, LIST_LABELS, REMOVE_LABEL, UPDATE_LABEL } from "../app/actionTypes";
-import { insert, remove, update } from "../utils/reducers";
+import { LIST_LABELS } from "../app/actionTypes";
 
 export const initialState = {
     documents: null,
 };
 
 export const labelsReducer = createReducer(initialState, builder => {
-    builder
-        .addCase(LIST_LABELS.SUCCEEDED, (state, action) => {
-            return { ...state, documents: action.payload };
-        })
-        .addCase(CREATE_LABEL.SUCCEEDED, (state, action) => {
-            return insert(state, action.payload);
-        })
-        .addCase(UPDATE_LABEL.SUCCEEDED, (state, action) => {
-            return update(state, action.payload);
-        })
-        .addCase(REMOVE_LABEL.SUCCEEDED, (state, action) => {
-            return remove(state, action.payload);
-        });
```

Comment:
Don't think this is going to be a major problem, but check what views are still using the redux labels list. With the current changes the redux list will no longer get updates, which could cause stale data to be rendered in views still using it.